pkgs/by-name/pd/pd/package.nix

```diff
@@ -0,0 +1,79 @@
+{ rev ? "a0f8550034958d71cb998bfc1d029bb02ff83320"
+, hash ? "sha256-0He3Ryqy4RxYuR2HbwGHJxegfOuKJ91B/JI+z2AK8wk="
+
+, lib
+
+, stdenv_32bit
+, fetchFromGitHub
+, fetchpatch
+
+, pkgsi686Linux # SDL2 and zlib
+
+, pkg-config
+, python3
+}: let
+  shortrev = builtins.substring 0 9 rev;
+in stdenv_32bit.mkDerivation rec {
```

Comment:
Is this perhaps a redundant rec? I couldn't spot where it was ever used (which is part of the reason why rec is frowned upon these days)

Reply:
It is! I'll remove it in my next commit